src/components/Links.tsx

```diff
@@ -31,15 +31,9 @@ interface TightLinkProps extends LinkProps {
   text: string
 }
 
-export const TightLinkStrict: React.FC<TightLinkProps> = ({ maxWidth, text, ...props }) => {
-  return (
-    <div style={{ maxWidth: maxWidth, display: 'flex', overflow: 'hidden' }}>
-      <StyledLink {...props} data-tip={text}>
-        {smartHash(text)}
-      </StyledLink>
-    </div>
-  )
-}
+export const SimpleLink: React.FC<LinkProps> = ({ children, ...props }) => (
```

Comment:
I know it's not your change, but we already import `FC` so no need to use it from `React`.

Reply:
Done!